### PR TITLE
Complete  mobile version

### PR DESCRIPTION
**Pull request by akbari4yaseen**
This branch contains:

- Toolbar
- Headline
- Works
- About Me
- Contact Me

### CODE REVIEW

File: portfolio.html [hunk 1]
```diff
@@ -20,46 +20,279 @@
...
+          </div>
+        </section>
+        <section class="works">
+          <article class="card bg-white">
+            <figure>
+              <img src="images/portfolio/tonic.png" alt="Tonic" />
+            </figure>
+            <div class="project-info">
+              <div class="project-header">
+                <h1 class="project-title">Tonic</h1>
```
Comment (mugishasam123) [inline portfolio.html]: - [ ] Kindly respect the order of headings in the HTML document to improve your page's accessibility. so you should have an `h1` tag in your document but other headings are allowed to be used multiple times. you got this 👍

File: portfolio.html [hunk 2]
```diff
@@ -20,46 +20,279 @@
...
+              </div>
+            </div>
+          </article>
+          <article class="card bg-white">
+            <figure>
+              <img src="images/portfolio/multi.png" alt="multi" />
+            </figure>
+            <div class="project-info">
+              <div class="project-header">
+                <h1 class="project-title">Multi-Post Stories</h1>
```
Comment (mugishasam123) [inline portfolio.html]: - [ ] Kindly use headings in proper way , so you can use other headings multiple times in the document except `h1`

File: portfolio.html [hunk 3]
```diff
@@ -20,46 +20,279 @@
...
+                <li class="tag">javaScript</li>
+              </ul>
+              <div class="action">
+                <button class="btn-default">See Project</button>
+              </div>
+            </div>
+          </article>
+        </section>
+        <section class="about bg-white">
+          <h1 class="title">About Myself</h1>
```
Comment (mugishasam123) [inline portfolio.html]: - [ ] Kindly use headings properly in your codes to make your work more professional. look more on [headings](https://www.w3schools.com/html/html_headings.asp)

**Review by mugishasam123 — CHANGES_REQUESTED** [review]
Hi @akbari4yaseen  👋 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
alt
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors 👍 
- Good commit messages 💯 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by victorokoroji — APPROVED** [review]
Hi @akbari4yaseen, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- Linters are working properly ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**